transcendence-app/src/user/infrastructure/db/postgres/block.postgres.repository.ts

```diff
@@ -42,4 +43,15 @@ export class BlockPostgresRepository
     });
     return blockData && blockData.length ? new this.ctor(blockData[0]) : null;
   }
+
+  async getBlocks(blockerId: string): Promise<User[] | null> {
+    const usersData = await makeQuery<UserData>(this.pool, {
```

Comment:
Why do we need to instantiate this with `UserData` instead of just plain `User`?

Reply:
We do this for readability. We type the query return to `UserData` because the result it's an object, then instantiate it to the `User` class for serialization.